src/query/service/src/servers/http/v1/session/login_handler.rs

```diff
@@ -34,8 +36,17 @@ struct LoginRequest {
 }
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
-struct LoginResponse {
+pub struct LoginResponse {
```

Comment:
should we change this to enum?
```
enum LoginResponse {
     Ok(...)
     Error(QueryError)
}
```

Reply:
version may be still helpful when error?

Reply:
done